doc/development/doxygen.rst

```diff
@@ -0,0 +1,6 @@
+.. _doxygen:
+
+C++ API documentation
+=====================
+
+The generated C++ API documentation can be found `here </doxygen/html/index.html>`__.
```

Comment:
This link doesn't work for me on my local file system, it turns into: `file:///doxygen/html/index.html` in my browser.

Reply:
Yepp, the way I linked to the site root here always needed a server. I now copy the files into the generate site with Sphinx (which might help with readthedocs as well), and used a relative link which should work both with and without a server.